sdk/openai/Azure.AI.OpenAI/src/Generated/AudioTranscriptionOptions.Serialization.cs

```diff
@@ -193,6 +201,59 @@ internal static AudioTranscriptionOptions DeserializeAudioTranscriptionOptions(J
                 serializedAdditionalRawData);
         }
 
+        private BinaryData SerializeMultipart(ModelReaderWriterOptions options)
+        {
+            using MultipartFormDataRequestContent content = ToMultipartRequestContent();
+            using MemoryStream stream = new MemoryStream();
+            content.WriteTo(stream);
+            if (stream.Position > int.MaxValue)
+            {
+                return BinaryData.FromStream(stream);
+            }
+            else
+            {
+                return new BinaryData(stream.GetBuffer().AsMemory(0, (int)stream.Position));
+            }
+        }
+
+        internal virtual MultipartFormDataRequestContent ToMultipartRequestContent()
+        {
+            MultipartFormDataRequestContent content = new MultipartFormDataRequestContent();
+            content.Add(AudioData, "file", "file");
```

Comment:
Why hard-code the filename instead of letting the end-user specify it?

Reply:
As we discuss about the `FileUpload` before, now typespec has no ideal to associate `FileName` meta-data to an `bytes`(which is hardcode interpret as a file in mutlipart/form-data). So codegen cannot figure out the property `FileName` is a file meta-data and associate it with file.  In openAI, we need to customize this.

Reply:
If there is no value for filename, the model should not pass a value to this parameter.

Reply:
We will not set default fileName for file if not define in spec. I will update this.